packages/atomic/cypress/e2e/search-interface.cypress.ts

```diff
@@ -261,7 +263,7 @@ describe('Search Interface Component', () => {
     it('should work with lowercase regions', () => {
       setLanguageAndWait('zh-tw');
 
-      QuerySummarySelectors.text().should('contain', '结果数');
```

Comment:
Funny enough, this test was actually passing, but was incorrect 😏

The setup is to test zh-tw and here it was displaying the simplified chinese version (ie: the bug this PR fixes)